modules/sharedaddy/sharing-service.php

```diff
@@ -749,7 +749,7 @@ function sharing_display( $text = '', $echo = false ) {
 	}
 
 	// Allow to be used on P2 ajax requests for latest posts.
-	if ( defined( 'DOING_AJAX' ) && DOING_AJAX && isset( $_REQUEST['action'] ) && 'get_latest_posts' == $_REQUEST['action'] ) {
+	if ( defined( 'DOING_AJAX' ) && DOING_AJAX && isset( $_REQUEST['action'] ) && apply_filters('sharing_ajax_action', 'get_latest_posts') == $_REQUEST['action'] ) {
```

Comment:
You'll also want to add spaces inside the parenthesis, as per the WordPress coding standards:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage